/**:
  ros__parameters:
    dt: 0.5
    is_axis_inverted: 1   # 1 or -1
    unwinding_factor: 1.0   # only affect the integral term. The higher the faster unwind. Set to -1 for instant unwind.

# Gentle
/**/y_pid:
  ros__parameters:
    kp: 0.015
    is_axis_inverted: -1
    input_topic: 'offset_position_y'
    output_topic: 'core/thrust_vector_cmd_horizontal/y'

/**/y_delta_pid:
  ros__parameters:
    kp: 0.5
    kd: 0.5
    is_axis_inverted: -1
    input_topic: 'offset_position_y_delta'
    output_topic: 'core/thrust_vector_cmd_horizontal/y_delta'

# Aggressive
/**/y_pid_turn:
  ros__parameters:
    kp: 0.04
    is_axis_inverted: -1
    input_topic: 'offset_position_y'
    output_topic: 'core/thrust_vector_cmd_horizontal/y'

/**/y_delta_pid_turn:
  ros__parameters:
    kp: 0.7
    kd: 0.5
    is_axis_inverted: -1
    input_topic: 'offset_position_y_delta'
    output_topic: 'core/thrust_vector_cmd_horizontal/y_delta'

# x_pid
/**/x_pid:
  ros__parameters:
    kp: 0.2
    ki: 0.005
    kd: 0.05
    unwinding_factor: 3.0
    input_topic: 'offset_position_x'
    output_topic: 'core/thruster_cmd_float'